docs/contributing/gauging_changes.md

```diff
@@ -0,0 +1,60 @@
+# Gauging changes
+
+A lot of the time, your change will affect formatting and/or performance. Quantifying
+these changes is hard, so we have tooling to help make it easier.
+
+It's recommended you take a look at quantifable changes your change will cause before
+submitting a PR if it seems disruptive enough.
```

Comment:
I found this hard to parse.
```suggestion
It's recommended you evaluate the quantifable changes your black formatting change
causes before submitting a PR. Think about if the change seems disruptive enough 
to cause frustration to projects that are already "black formatted".
```

Reply:
I tend to either write too densely or with too many words, this seems wayyy more reasonable. Thanks!

Reply:
Also quantifable -> quantifiable